scripts/lint-dividing-lines.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if grep -Ernxo -e "/{4,79}" -e "/{81,}" ./src/; then
```

Comment:
Why 80? This sounds like a setting from the times we had 15'' monitors.
120 would be more realistic.

Reply:
120 used as a base value -- done

Reply:
Added https://github.com/kamu-data/kamu-cli/pull/685/files#diff-ac447473973f0a9697d6262d62ac648b6563dfffdb827e2bd7531dee2f0b8932

Reply:
Rust [style guide](https://doc.rust-lang.org/nightly/style-guide/) says:

`The maximum width for a line is 100 characters`

I wouldn't care if its 77 or 89 as long as its consistent everywhere, but wouldn't setting it above the limit we use for `rustfmt` lead to problems?